db/migrate/20211116054502_create_member_groups.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class CreateMemberGroups < ActiveRecord::Migration[5.2]
+  DEFAULT_RATES_LIMITS = { 'second' => 10, 'minit' => 100 }.freeze
+
+  OPEN_ORDERS_LIMITS = {
+    'market-makers' => 50,
+    'vip-3' => 5,
+    'vip-0' => 4, # bargainer on sandbox
+    'default' => 2
+  }.freeze
+
+  def open_orders_limit
+    member_group.open_orders_limit
+    OPEN_ORDERS_LIMITS.fetch(group, OPEN_ORDERS_LIMITS.fetch('default', 1))
+  end
+
+  def rates_limits
+    { 'second' => 100, 'minut' => 1000 }
+  end
+
+  def change
+    create_table :member_groups do |t|
+      t.string :key, length: 10, null: false
```

Comment:
```suggestion
      t.string :key, limit: 10, null: false
```
тут, наверно, limit имелся в виду?